frontend/src/metabase/nav/components/AppBar/AppBarLarge.styled.tsx

```diff
@@ -44,6 +46,31 @@ export const AppBarLeftContainer = styled.div<AppBarLeftContainerProps>`
       opacity: ${props => (props.isNavBarEnabled ? 1 : 0)};
     }
   }
+
+  ${props =>
+    !props.isLogoVisible &&
+    css`
+      ${SidebarButton} {
+        opacity: 1;
+      }
+
+      ${SidebarIcon} {
+        color: ${color("text-medium")};
```

Comment:
Previously it wasn't possible to see the toggle button without hovering on desktop, so we didn't have the non-hovered state. But this could happen after we disabled the logo.
https://metaboat.slack.com/archives/C045W5MF84U/p1671814108759669?thread_ts=1671734932.320549&cid=C045W5MF84U